exercises/series/description.md

```diff
@@ -1,16 +1,16 @@
 Given a string of digits, output all the contiguous substrings of length `n` in
-that string.
+that string in the order that they appear.
 
 For example, the string "49142" has the following 3-digit series:
 
-- 491
-- 914
-- 142
+- "491"
+- "914"
+- "142"
 
 And the following 4-digit series:
 
-- 4914
-- 9142
+- "4914"
+- "9142"
 
 And if you ask for a 6-digit series from a 5-digit string, you deserve
 whatever you get.
```

Comment:
I interpret this last statement to mean that each track can decide how to deal with potentially erroneous input.  I know that has been debated to an extent in this PR.  If the consent were to treat erroneous a certain way then this statement should probably be reworded to reflect that decision.